cuegui/cuegui/MainWindow.py

```diff
@@ -422,3 +429,37 @@ def __saveSettings(self):
                                self.size())
         self.settings.setValue("%s/Position" % self.name,
                                self.pos())
+
+    def __removeConfig(self, config_file):
+        try: 
```

Comment:
Could you explain this chunk of code? I'm having a hard time following it.

Reply:
we fork the process and let the parent process shutdown, the child then sleep for 3 seconds and remove the config file.
If we remove the config file before saving it's overwritten by Qsettings on closure. this is a very Unix specific hack on our part.

Reply:
Ok thanks, I see now. A quick search didn't turn up a way to disable this QSettings behavior, so that makes sense.

What if instead we saved a "setting" that actually means "restore the default settings on next startup"? Then in `Main.startup` we could change the check to be "if local settings don't exist or they contain the overwrite-me setting, then copy the default config files into place".